include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -44,19 +44,139 @@ using non_void_type = typename ::std::enable_if<!::std::is_void<_Tp>::value, _Tp
 //std::logical_and and std::logical_or are not supported in Intel(R) oneAPI DPC++ Compiler to be used in sycl::inclusive_scan_over_group and sycl::reduce_over_group
 template <typename _BinaryOp, typename _Tp>
 using __has_known_identity =
```

Comment:
Proposed implementation of ```__has_known_identity``` according to https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.identities